pliers/extractors/base.py

```diff
@@ -82,7 +82,8 @@ def __init__(self, data, stim, extractor, features=None, onsets=None,
             durations = stim.duration
         self.durations = durations if durations is not None else np.nan
 
-    def to_df(self, timing=True, metadata=False):
+    def to_df(self, timing=True, metadata=False, format='wide',
```

Comment:
I think the default `format` here should be the same as that of `merge_results`. That being said, still more in favor of defaulting to `wide`.